internal/support/scheduler/router.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2021 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package scheduler
+
+import (
+	"net/http"
+
+	"github.com/edgexfoundry/go-mod-bootstrap/v2/bootstrap/container"
+	"github.com/edgexfoundry/go-mod-bootstrap/v2/di"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/common"
+	"github.com/gorilla/mux"
+
+	commonController "github.com/edgexfoundry/edgex-go/internal/pkg/controller/http"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/correlation"
+	schedulerController "github.com/edgexfoundry/edgex-go/internal/support/scheduler/controller/http"
+)
+
+func LoadRestRoutes(r *mux.Router, dic *di.Container) {
+	// v2 API routes
+	// Common
+	cc := commonController.NewV2CommonController(dic)
```

Comment:
Remove `V2` from controller name. Same for all controllers with `V2` in the name.

Reply:
Removed.